surfer/viz.py

```diff
@@ -1737,6 +1748,45 @@ def add_text(self, x, y, text, name, color=None, opacity=1.0,
         if justification is not None:
             text.property.justification = justification
 
+    def add_text3d(self, x, y, z, text, name, color=None, opacity=1.0,
+                 row=-1, col=-1, font_size=None, justification=None):
+        """ Add a text to the visualization
+
+        Parameters
+        ----------
+        x : Float
+            x coordinate
+        y : Float
+            y coordinate
+        z : Float
+            z coordinate
+        text : str
+            Text to add
+        name : str
+            Name of the text (text label can be updated using update_text())
+        color : Tuple
+            Color of the text. Default is the foreground color set during
+            initialization (default is black or white depending on the
+            background color).
+        opacity : Float
+            Opacity of the text. Default: 1.0
+        row : int
+            Row index of which brain to use
+        col : int
+            Column index of which brain to use
+        """
+        if name in self.texts_dict:
+            self.texts_dict[name]['text'].remove()
+        text = self.brain_matrix[row, col].add_text3d(x, y, z, text,
+                                                    name, color, opacity)
+        self.texts_dict[name] = dict(row=row, col=col, text=text)
+        if font_size is not None:
+            text.property.font_size = font_size
+            text.actor.text_scale_mode = 'viewport'
+        if justification is not None:
```

Comment:
This parameter doesn't appear in the docstring. Also despite it being the property name in mayavi, "justification" is a weird term and "alignment" would be more correct.